TUTORIAL.md

```diff
@@ -0,0 +1,92 @@
+# Guide to Solving and Reviewing Tweet Shortener
+
+Before we start coding our methods, we need to create a hash where our methods can look up which word needs to be substituted.
+
+```ruby
+def dictionary
+  dictionary = {
+    "too" => "2",
+    "to" => "2",
+    "two" =>"2",
+    "four" => "4",
+    "for" => "4",
+    "be" => "b",
+    "you" => "u",
+    "at" => "@",
+    "and" => "&"
+  }
+end
+```
+
+### `word_substituter`
+
+Let us start with defining our `word_substituter` method. This method will also take in an argument `tweet`.
+
+```ruby
+def word_substituter(tweet)
+end
+```
+After putting a `binding.pry` in our method, we know that the passed in `tweet` is a sting.
```

Comment:
:+1: 

(can you show them what you mean by "putting a `binding.pry` in our method?)